esp32-hal/examples/i2c_display.rs

```diff
@@ -49,7 +50,7 @@ fn main() -> ! {
         peripherals.I2C0,
         io.pins.gpio32,
         io.pins.gpio33,
-        100_000,
+        HertzU32::kHz(100),
```

Comment:
Does the extension trait not work here? Can we do `100.khz()` instead?

Reply:
Yes, we can. I just wanted to be more persistent (because of `CONST`s).

Reply:
I believe you need to specify the type in order to use the extension trait, but this is already being done in the `spi_loopback` examples: `100u32.kHz()` for example. I would prefer this form be used, personally.